src/pages/post.js

```diff
@@ -33,6 +33,7 @@ const Post = ({ match }) => {
 
   return (
     <>
+      <MetaTags post={ currentPost } />
```

Comment:
Same function call, but we pass para ... same Component rendered, but we pass props.